File: WebApp/Scripts/app/model/UR_ID.js
Ext.define('WEBAPP.model.UR_ID', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'TUSER', type: 'string' },        // 人員代碼
        { name: 'INID', type: 'string' },         // 責任中心代碼
        { name: 'PA', type: 'string' },           
        { name: 'SL', type: 'string' },           // SALT
        { name: 'UNA', type: 'string' },          // 中文姓名
        { name: 'UENA', type: 'string' },         // 英文姓名
        { name: 'IDDESC', type: 'string' },       // 備註
        { name: 'EMAIL', type: 'string' },        // EMAIL
        { name: 'EXT', type: 'string' },          // 分機
        { name: 'BOSS', type: 'string' },         // 是否主管
        { name: 'TITLE', type: 'string' },        // 職稱
        { name: 'FAX', type: 'string' },          // 傳真
        { name: 'FL', type: 'string' },           // 是否有效(1有效0無效)
        { name: 'TEL', type: 'string' },          // 電話
        { name: 'CREATE_TIME', type: 'string' },  // 建立時間
        { name: 'CREATE_USER', type: 'string' },  // 建立人員代碼
        { name: 'UPDATE_TIME', type: 'string' },  // 異動時間
        { name: 'UPDATE_USER', type: 'string' },  // 異動人員代碼
        { name: 'UPDATE_IP', type: 'string' }     // 異動IP
    ]
});

File: WebApp/Models/UR/UR_ID.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class UR_ID : JCLib.Mvc.BaseModel
    {
        public string TUSER { get; set; }
        //public string DEPT_NO { get; set; }
        public string INID { get; set; }
        public string PA { get; set; }
        public string SL { get; set; }
        public string UNA { get; set; }
        public string IDDESC { get; set; }
        public string EMAIL { get; set; }
        public string EXT { get; set; }
        public string BOSS { get; set; }
        public string TITLE { get; set; }
        public string FAX { get; set; }
        public Nullable<decimal> FL { get; set; }
        public string TEL { get; set; }
        public string ADUSER { get; set; }
        public string INID_NAME { get; set; }
        public string CREATE_TIME { get; set; }
        public string CREATE_USER { get; set; }
        public string UPDATE_TIME { get; set; }
        public string UPDATE_USER { get; set; }
        public string UPDATE_IP { get; set; }
        public string WHITELIST_IP1 { get; set; }
        public string WHITELIST_IP2 { get; set; }
        public string WHITELIST_IP3 { get; set; }
        public string EM_EMPNAME { get; set; }
        public string EM_DEPTNO { get; set; }
    }
}